pkgs/development/python-modules/django-pydantic-field/default.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  django,
+  pydantic,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "django-pydantic-field";
+  version = "v0.3.12";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "surenkov";
+    repo = "django-pydantic-field";
+    rev = version;
+    hash = "sha256-rlnS67OGljWD8Sbyutb43txAH0jA2+8ju1ntSEP3whM=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
+
+  propagatedBuildInputs = [
+    django
+    pydantic
+  ];
+
+  meta = with lib; {
+    description = "Django JSONField with Pydantic models as a Schema";
```

Allows automated updates to embed the changelog URL, which in turn makes reviews faster.

```suggestion
    changelog = "https://github.com/surenkov/django-pydantic-field/releases/tag/${src.tag}";
    description = "Django JSONField with Pydantic models as a Schema";
```